po/hu.po

```diff
@@ -0,0 +1,214 @@
+# Hungarian translation
+# Copyright (C) 2022 Kokan <kokaipeter@gmail.com>
```

Comment:
This is the copyright for the project, not for the translation author :). Leave it as it was.

Reply:
Sure.